src/modules/auth/components/CustomDerivationPath/index.js

```diff
@@ -1,12 +1,23 @@
-import React from 'react';
+import React, { useMemo, useState } from 'react';
 import { useTranslation } from 'react-i18next';
+import { getDerivationPathErrorMessage } from 'src/modules/wallet/utils/account';
 import { Input } from 'src/theme';
+import { defaultDerivationPath } from 'src/utils/explicitBipKeyDerivation';
 
-const CustomDerivationPath = ({ onChange, value, errorMessage }) => {
+const CustomDerivationPath = ({ onChange }) => {
   const { t } = useTranslation();
 
-  const onPathInputChange = (e) => {
-    onChange(e.target.value);
+  const [derivationPath, setDerivationPath] = useState(defaultDerivationPath);
+  const derivationPathErrorMessage = useMemo(
+    () => getDerivationPathErrorMessage(derivationPath),
+    [derivationPath]
+  );
+
+  const onPathInputChange = ({ target }) => {
+    const value = target.value;
+
+    setDerivationPath(value);
+    onChange?.({ derivationPathErrorMessage, derivationPath: value });
```

Comment:
To fix above we need to do
`onChange?.({ derivationPathErrorMessage:  getDerivationPathErrorMessage(derivationPath), derivationPath: value });`

But i really dont recommend it because "duplicate vars holding same data" bugs are real ninjas in hiding. 
We cannot encapsulate derivation path logic here since parent need it.

Reply:
If we had a `FieldSet` component this component wouldn't even be needed. The only reason it is called CustomDerivationPath right now is because of the label text. All logic is handled in parents